client/stylesheets/_responsive.scss

```diff
@@ -298,6 +293,20 @@
 }
 
 @media (max-width: 650px) {
+    .profileCardInfo li:first-child {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `profileCardInfo` should be written in lowercase with hyphens